### PR TITLE
Update structure-of-a-contract.rst

### PR DESCRIPTION
**Pull request by aj07**

### CODE REVIEW

**Comment by aj07** [comment]
https://github.com/ethereum/solidity/issues/3541

https://github.com/ethereum/solidity/pull/3967

File: docs/structure-of-a-contract.rst [hunk 1]
```diff
@@ -109,7 +109,7 @@ Struct Types
 =============
 
 Structs are custom defined types that can group several variables (see
-:ref:`structs` in types section).
+:ref:`structs` in types section) and can be accessed by single name
```
Comment (chriseth) [inline docs/structure-of-a-contract.rst:112]: Please use full stops to end sentences.

Do you want to highlight "single" in any way?

File: docs/structure-of-a-contract.rst [hunk 2]
```diff
@@ -123,6 +123,16 @@ Structs are custom defined types that can group several variables (see
             uint vote;
         }
     }
+    
+If you want to create a new struct variable inside a function,
```
Comment (chriseth) [inline docs/structure-of-a-contract.rst:127]: I think this should rather go here: https://solidity.readthedocs.io/en/v0.4.23/types.html#structs

File: docs/structure-of-a-contract.rst [hunk 3]
```diff
@@ -123,6 +123,16 @@ Structs are custom defined types that can group several variables (see
             uint vote;
         }
     }
+    
+If you want to create a new struct variable inside a function, 
+and then add push it to an array, then you should create it in
```
Comment (chriseth) [inline docs/structure-of-a-contract.rst:128]: I would not mention whether the struct variable is created inside a function, but rather only mention the intention to add a new element to a struct array.

File: docs/structure-of-a-contract.rst [hunk 4]
```diff
@@ -123,6 +123,16 @@ Structs are custom defined types that can group several variables (see
             uint vote;
         }
     }
+    
+If you want to create a new struct variable inside a function, 
+and then add push it to an array, then you should create it in 
+the storage space not memory space ::
+
+    MyStruct memory newStruct;
```
Comment (chriseth) [inline docs/structure-of-a-contract.rst:131]: This example actually creates a memory object.

**Comment by chriseth** [comment]
I don't see any value in keeping this pull request open.

**Comment by aj07** [comment]
@chriseth Please keep this open. Next updated patch would be sent here. Else I would have to create new patch

**Comment by chriseth** [comment]
@aj07 pleise either reopen or create a new one if you add something here.

**Pull request closed without merging**